scripts/globals/job_utils/dancer.lua

```diff
@@ -202,6 +235,69 @@ xi.job_utils.dancer.useReverseFlourishAbility = function(player, target, ability
     return tpGained
 end
 
+xi.job_utils.dancer.checkWildFlourishAbility = function(player, target, ability)
+    if player:getAnimation() ~= 1 then
+        return xi.msg.basic.REQUIRES_COMBAT, 0
+    else
+        if player:hasStatusEffect(xi.effect.FINISHING_MOVE_1) then
+            return 0, 0
+        else
+            return xi.msg.basic.NO_FINISHINGMOVES, 0
+        end
+    end
```

Comment:
Why has the logic for all the different Finishing Moves Status effects been removed?

Reply:
It seems like whoever had set these job_utils up before had made it so instead of having the 6 different finishing move statuses each be shown using their own effect, they just put the finishing move count in to the "power" of the first one's effect. I just confirmed to that logic and brought the new abilities in. If the goal is to move back to the old way of doing it with each finishing move having its own status, that's a great idea, but I think outside of the scope of this PR as it's just meant to bring in functionality that's missing for now.